app/components/footer.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link } from "@remix-run/react";
+import PrivacyPolicy from "~/routes/privacy_policy";
+
+import ReactDenverLogo from "../assets/react_denver_logo.svg";
+import YouTube from "../assets/YouTube.svg";
+import Discord from "../assets/Discord.svg";
+import GitHub from "../assets/GitHub.svg";
+import Twitter from "../assets/Twitter.svg";
```

Comment:
Have a look at Remix's directory import paths. I think by default it's `~/` so instead of going relative path down to root and into assets it would look more like `import YouTube from "~/assets/YouTube.svg"` Just incase you ever want to move a file around and you don't get into a relative path nightmare. Ie. `../../../thing/thing2/foo.tsx`